Tests/govuk_ios/govuk_ios_unit_tests/Specs/Extensions/Container/Container+APIClientsTests.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import Testing
+
+import Factory
+
+@testable import govuk_ios
+
+@Suite
+struct Container_APIClientTests {
+    @Test
+    func searchAPIClient_returnsExpectedResult() async {
```

Comment:
I'd be inclined to change the name of this test to searchAPIClient_createsExpectedRequest.  There is nothing about the result being tested.

Reply:
Updated 👍 Thanks!